app/bingey-ui/src/components/App/App.jsx

```diff
@@ -11,7 +11,7 @@ export const App = () => {
     <>
       <Router>
         <Switch>
-          <Route exact path='/admin'>
+          <Route path='/admin'>
```

Comment:
This doesn't seem related, why is this here?

Reply:
My bad, I forgot to comment on this. Found a bug where refreshing on certain admin tabs would not render the admin panel but the main page, this fixes it.